dashboard/app/controllers/api_controller.rb

```diff
@@ -83,7 +83,7 @@ def section_progress
     # student level completion data
     students = @section.students.map do |student|
       level_map = student.user_levels_by_level(@script)
-      paired_user_level_ids = PairedUserLevel.pairs(level_map.keys)
+      paired_user_level_ids = PairedUserLevel.pairs(level_map.values.map(&:id))
```

Comment:
Note that `level_map` is a hash from `level_id`s to `UserLevel`s. And that `PairedUserLevel.pairs` accepts a collection of `user_level_ids`.